docs/pages/tutorial/eas/configure-development-build.mdx

```diff
@@ -0,0 +1,220 @@
+
+## Understanding development builds
+
+Let's start by learning about what are development builds and why we need them.
+
+A [development build](/develop/development-builds/introduction/) is a debug version of our project. It is optimized for quick iterations when creating an app. Using Expo means including the [`expo-dev-client`](/versions/latest/sdk/dev-client/) library, which offers a robust and complete development environment. This setup allows us to integrate any native library or change code inside the [native directories](/workflow/overview/#android-and-ios-native-projects) as required.
+
+## Benefits of creating a development build
+
+> **Note:** If you are familiar with [Expo Go](/get-started/expo-go/), think of a development build as a customizable version of Expo Go that is unique to the requirements of a project.
+
+A development build offers several advantages. Here are some key highlights in comparison to Expo Go:
+
+| Feature                           | Development Builds                                                                                                                             | Expo Go                                                                                              |
+| --------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| **Development phase**             | Offers web-like iteration speed for mobile app development.                                                                                    | Allows for quick iteration and testing of Expo SDK projects using the client app.                    |
+| **Collaboration**                 | Facilitates team testing with shared native runtime.                                                                                           | Easy project sharing via QR codes on a device.                                                       |
+| **Third-party libraries support** | Full support for any [third-party library](/workflow/using-libraries/#third-party-libraries), including those that require custom native code. | Limited to libraries within the Expo SDK, not suitable for custom native dependencies.               |
+| **Customization**                 | Extensive customization with [config plugins](/config-plugins/introduction/) and direct access to native code.                                 | Limited customization with a focus on Expo SDK capabilities without direct native code modification. |
+| **App size**                      | Optimized app size, including only the libraries used in the project.                                                                          | Generally larger due to the inclusion of the entire Expo SDK.                                        |
```

Comment:
```suggestion
```

Reply:
this isn't really useful information. app size doesn't matter for development builds. in fact, your development build might even be larger than expo go distributed through the app store/play store because it will bit split for your specific device when installed through the store